docs/catalyst_101.md

```diff
@@ -136,22 +136,55 @@ Once the existing members have voted to offer you membership, it’s up to you t
 
 ## Project Management
 
-### Tools
```

Comment:
I think this tool information is still useful for new folks. What was your rational for removing it?

Reply:
I definitely wanted to remove the ZenHub language, and I didn't feel like these fit under "Project management" - I think it makes sense to pull them out into their own Communication + Admin Tools sections. It's useful to know what we intend to do with each of our many tools.